open-craft/opencraft
# SE-589 Fix encode utf-8 values in instance models

This PR fix an issue when OCIM is being created with special caracteres in instance name
Example: Faculdade de S**ã**o Paulo

**JIRA tickets**: [SE-589](https://tasks.opencraft.com/browse/SE-589)

**Merge deadline**: "None" 

**Testing instructions**:

1. Clone this branch of OCIM
2. Execute vagrant up and vagrant ssh command
3. Execute make test.unit
4. Should execute without problem


**Author notes and concerns**:

1. Currently looking for test in OCIM (@symbolist I will help in next week)
2. I added a new `assert` at tests

**Reviewers**
